### PR TITLE
issue #124 Email is checked in lower case.

### PR DESCRIPTION
**Pull request by geekdenz**
This is because email addresses are case insensitive by standard.

### CODE REVIEW

**Comment by nglasl** [comment]
I've run out of time today, but I will look at this for you first thing tomorrow morning.

File: code/forms/MemberProfileValidator.php [hunk 1]
```diff
@@ -45,7 +45,13 @@ public function php($data) {
 				sprintf('"%s" = \'%s\'', Convert::raw2sql($field), Convert::raw2sql($data[$field]))
 			);
 
-			if ($other && (!$this->member || !$this->member->exists() || $other->ID != $this->member->ID)) {
+			$isEmail = $field === 'Email';
+			$emailOK = !$isEmail;
+			if ($isEmail) {
+				$member = Member::get()->filter('Email:nocase', strtolower($data['Email']))->first();
```
Comment (nglasl) [inline code/forms/MemberProfileValidator.php]: If I understand the documentation correctly, you probably no longer need the `strtolower` here. Can you please test and confirm whether that's the case? I'm happy to merge once that's done.

Comment (geekdenz) [inline code/forms/MemberProfileValidator.php]: You are correct. However, we have tested the changes like this now, so please accept the pull request and change this yourself. Functionally and mathematically the result should be the same but technically we would have to do the tests again regardless. This is an expensive exercise and not worth the saving of a few nano seconds of processing time. String functions are very fast. 

So, please accept and change yourself if you have the time and can be bothered. It definitely fixes me than it breaks like it is now.

**Comment by rasstislav** [comment]
After this change, I am not able to save my profile. Error message: There is already a member with the same ...

**Comment by nglasl** [comment]
Seems this was an oversight. I have pushed a fix (@rasstislav), so please let me know if you're still having problems.

**Comment by geekdenz** [comment]
@nglasl What is the fix as we need it as well.

**Comment by nglasl** [comment]
https://github.com/silverstripe-australia/silverstripe-memberprofiles/commit/ce98c97037dd6ab1ff6e3fb4ab7386c2377febc5

**Comment by rasstislav** [comment]
@nglasl it works, thanks.

**Comment by geekdenz** [comment]
FYI: Just learnt that I can merge a specific commit with
```
git cherry-pick <commit hash>
```
In this case:
```
git cherry-pick ce98c97037
```

**Pull request merged**